### PR TITLE
Initial commit of multiple python versions

### PR DESCRIPTION
**Pull request by dwoz**
This work still needs a fair amount of re-factoring but it's a start. Commiting it now so we can get some testing going.

### CODE REVIEW

**Comment by dwoz** [comment]
The windows failure on python 3.11.1 are related to this:

https://github.com/python/cpython/issues/100320

I am testing to see if it's resolved in 3.11.2

**Comment by dwoz** [comment]
3.11.2 works for windows.. turns out it's also working everywhere else! :)

File: relenv/build/common.py [hunk 1]
```diff
@@ -72,6 +71,8 @@
 for __path in sys.path:
     if __path.startswith(__valid_path_prefixes):
         __sys_path.append(__path)
+for __path in os.environ.get('PYTHONPATH', '').split(':'):
```
Comment (s0undt3ch) [inline relenv/build/common.py:74]: Does tbis code run on windows?

**Review by s0undt3ch — APPROVED** [review]

File: relenv/build/common.py (hunk 1 above)
Comment (dwoz) [reply relenv/build/common.py:74]: Yes, I'll tackle this in a follow up pr

**Comment by dwoz** [comment]
This PR helps address #75

**Comment by dwoz** [comment]
Adresses #7

**Pull request merged**